doc/release_notes/release_notes_2.5.rst

```diff
@@ -57,93 +57,67 @@ Upgrading to v2.5 From Previous Releases
+ - :option:`vm.PTM`
 
-   $ sudo apt install libxml2-utils xsltproc
-   $ sudo pip3 install lxml xmlschema
+The following element is renamed:
 
-.. note::
-   This is not the complete list of tools required to build ACRN. Refer to
-   :ref:`getting-started-building` for a complete guide to get started from
-   scratch.
+ - :option:`hv.FEATURES.SSRAM.SSRAM_ENABLED` (was ``hv.FEATURES.PSRAM.PSRAM_ENABLED`` in v2.4)
 
-Configuration File Format
-=========================
+Constraints on values of the following element is changed:
 
-Starting with release v2.4, Kconfig is no longer used, and the contents of
-scenario XML files have been simplified. If you're using v2.3 or earlier, you
-need to upgrade your own Kconfig-format files or scenario XML files if you
-maintain any.
+ - :option:`vm.guest_flags.guest_flag` no longer accepts an empty text. For VMs
+   without any guest flag set, a ``0`` should be given.
 
-See the instructions in the :ref:`release_notes_2.4` release notes for helpful
-instructions to assist in this upgrade.
+To upgrade a v2.4 compliant scenario XML to v2.5, run the following command on
+the host machine:
 
+.. code-block:: bash
 
-Build Commands
-==============
-
-We recommend you update the usage of variables ``BOARD_FILE`` and
-``SCENARIO_FILE``, which are being deprecated,  and ``RELEASE``:
-
- - ``BOARD_FILE`` should be replaced with ``BOARD``. You should not specify
-   ``BOARD`` and ``BOARD_FILE`` at the same time.
- - Similarly, ``SCENARIO_FILE`` should be replaced with ``SCENARIO``.
- - The value of ``RELEASE`` should be either ``y`` (previously was ``1``) or
-   ``n`` (previously was ``0``).
+   $ misc/config_tools/upgrade_scenario_xml <path to scenario XML> v2.4
 
-``BOARD_FILE`` and ``SCENARIO_FILE`` can still be used but will take effect
-only if ``BOARD`` and ``SCENARIO`` are not defined. They will be deprecated in
-a future release.
+The script will generate a v2.5 compliant scneario XML by applying the changes
+above to the given one. The ``v2.4`` in the command above is the version of your
+scenario XML. If you have a v2.3 compliant scenario XML, you can then upgrade
+using:
 
-Patches on Generated Sources
-============================
+.. code-block:: bash
 
-The C files generated from board and scenario XML files were removed from the
-repository in v2.4. Instead they will be generated in the build output when
-building the hypervisor. See the instructions in the :ref:`release_notes_2.4`
-release notes for more information.
+   $ misc/config_tools/upgrade_scenario_xml <path to scenario XML> v2.3
 
-Modifying generated files is not a recommended practice.
-If you find a configuration that is not flexible enough to meet your
-needs, please let us know by sending mail to `the acrn-dev mailing
-list <https://lists.projectacrn.org/g/acrn-dev>`_ or submitting a
-`GitHub issue <https://github.com/projectacrn/acrn-hypervisor/issues>`_.
+Upgrades from scenario XMLs prior to v2.3 is not supported.
```

Comment:
This seems a rather important note, and should be put at the front of this section not at the end.

Reply:
I tried to shorten the description of the usage of the script a bit in v2, but find no way to put this note at the front where the script itself is not introduced. I used a note to emphasis it from normal paragraphs instead.